ui.frontend/src/main/webpack/site/js/buildSearch.ts

```diff
@@ -20,7 +20,27 @@ export const buildSearch = async (
     return
   }
 
-  const { callbacks, autoSuggestionDebounceTime = 500 } = options || {}
+  const { trackingUrl } = searchConfig
+
+  const { callbacks = {}, autoSuggestionDebounceTime = 500 } = options || {}
+
+  if (trackingUrl) {
+    callbacks.onSearchItemClick = (url: string) => {
+      const headers = new Headers()
+      headers.append('Content-Type', 'application/x-www-form-urlencoded')
+      fetch(trackingUrl, {
+        method: 'POST',
+        headers,
+        body: `trackedUrl=${url}`,
+      })
+        .catch((error) => {
+          console.error(`Error while tracking search result item with url: ${url}.`, error)
+        })
+          .finally(() => {
+            window.location.href = url
```

Comment:
mmh wait, isn't this  happening anyhow? need your input here @kevinolivar

Reply:
yes when we click on an item it should already redirect. Don't think 'finally' is needed

Reply:
@nhirrle I thought so too, but it is not the case. that `event.preventDefault()` that @kevinolivar mentioned is the issue i think

Reply:
ok thanks. well when we have the events ticket enabled we can remove this click listener again and checnge instead ot listen on the event